### ISSUE TITLE
%20 in Python installation filename leads to pip install error

### ISSUE THREAD

**Issue by cephalin** [projectkudu/kudu#2682]
This is related to https://github.com/Azure/azure-python-siteextensions/issues/18 which was closed but the issue remains. I'm guessing that the issue exists in all Python packages installed as site extensions. Please see there for error behavior and shell output.

@zooba as worked on it in https://github.com/Azure/azure-python-siteextensions/commit/edd6be5a24ed35bb45faa7bb9aa831683c7ce54d, but the path for the script fix is not quite correct. I think the offending line is:

```
ren D:\home\%SRC%\Lib\site-packages\setuptools\launcher%%%%20manifest.xml "launcher manifest.xml"

```

Whereas it probably should be 

```
ren D:\home\%SRC%\Lib\site-packages\setuptools\command\launcher%20manifest.xml "launcher manifest.xml"
```

**Comment by davidebbo** [comment]
I cannot repro this. When installing this package, I end up with `D:\home\python362x86\Lib\site-packages\setuptools\command\launcher%20manifest.xml`. Now it could be argued that it should instead be `D:\home\python362x86\Lib\site-packages\setuptools\command\launcher manifest.xml`, but that is not at all what you describe.

**Comment by cephalin** [comment]
Sorry I pasted from the email trail without too much thought. I was analyzing why the fix didn't fix the issue and was suggesting how the script might need to be modified. But yes the end result should be that the file is renamed to `D:\home\python362x86\Lib\site-packages\setuptools\command\launcher manifest.xml`.

**Comment by davidebbo** [comment]
But how do you end up with `launcher%%%%20manifest.xml` with 4 % signs? Or do you mean that you don't end up with that after all, but you see what I describe in my first reply?

**Comment by cephalin** [comment]
I'm saying that's what https://github.com/Azure/azure-python-siteextensions/commit/edd6be5a24ed35bb45faa7bb9aa831683c7ce54d has.

**Comment by cephalin** [comment]
BTW I've fixed the descriptions.

**Comment by davidebbo** [comment]
Adding @zooba. I don't understand quite what this script is doing. The one fix we can make in Kudu is that if the nupkg has `foo%20bar` in a file name, it will end up as `foo bar` on disk. Once that happens, I would not think that any script should be dealing with %20's at all.

Is what I describe the fix that we'd like to have in Kudu?

**Comment by davidebbo** [comment]
Fixed in b5c441a3ef34fcba103ecc4664fc3f94c65c6d5e (pending deployment to Azure).
